components/brave_ads/browser/ads_tab_helper.cc

```diff
@@ -110,10 +110,6 @@ void AdsTabHelper::DidFinishLoad(
   TabUpdated();
 }
 
-void AdsTabHelper::DidAttachInterstitialPage() {
```

Comment:
Could you please let me know why this was removed (or was it just deprecated in latest bump)? Thanks

Reply:
It was removed in Chromium bump: https://chromium.googlesource.com/chromium/src/+/07f282fe180ae29ba07000510b709bf3a358001b.